react-frontend/src/components/CaseStudies/horizontalinfocard.js

```diff
@@ -11,7 +11,7 @@ const HorizontalInfoCard = (props) => {
       bordered={false}
       cover={
         <div>
-          <img className="cardImageHorizontal" alt="example" src={props.img} />
+          <img className="cardImageHorizontal" alt="" src={props.img} />
```

Comment:
should probably be the title of the case study

Reply:
The ticket explicitly requires it be null. (for accessibility standards)